test/beacon/Voucher.test.ts

```diff
@@ -1051,6 +1053,33 @@ describe('Voucher', function () {
         });
     });
 
+    describe('Drain', async function () {
+        it('Should drain RLC balance of voucher', async function () {
```

Comment:
We only see RLC in this file => In comments and in the test name => otherwise we see every else SRLC

Reply:
I honestly don't think that the difference is much important here (and can be confusing) that's why I've chosen the simpler version RLC. But if you think otherwise I can do the renaming in a separate PR to not affect the size of this one.

Reply:
I think Gabriel is right here, you are better not mentioning RLC and use something like "balance on Poco", but if you introduce the token name you should use SRLC

Reply:
I agree you should add a todo for mentions that are not part of this PR :+1: